tests/tests.py

```diff
@@ -36,6 +36,7 @@
 TEST_INTPUT_MULTI = "multi_ori_image.jpg"
 TEST_INPUT_MULTI_CONTOUR = "roi_objects.npz"
 TEST_INPUT_ClUSTER_CONTOUR = "clusters_i.npz"
+TEST_INPUT_GENOTXT="cluster_names.txt"
```

Comment:
PEP8 recommends spaces around operators like `=`

Reply:
Unless they are in function calls...

Reply:
fixed it!